crates/nostr/src/event/tag/mod.rs

```diff
@@ -298,12 +298,15 @@ impl Tag {
     }
 
     /// Compose `["t", "<hashtag>"]` tag
+    ///
+    /// This will convert the hashtag to lowercase.
     #[inline]
     pub fn hashtag<T>(hashtag: T) -> Self
     where
         T: Into<String>,
```

Comment:
Since we use the `to_lowercase` later, here we can use `AsRef<str>`, to avoid the (possible) unnecessary conversion from `&str` to `String` (`to_lowercase` is implemented on `str` type).